uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-balance-tab-filters.component.spec.js

```diff
@@ -116,7 +116,7 @@ define([
         });
 
         it('emits a filters updated event with the selected filter values', function () {
```

Comment:
I think that could be better for readability: 'emits a "filters updated" event with the selected filter values'